application/cms/models.py

```diff
@@ -575,6 +579,19 @@ def review_token_expires_in(self, config):
         except Exception:
             return 0
 
+    @property
+    def has_known_statistical_errors(self) -> bool:
+        return any(
+            later_minor_version.update_corrects_data_mistake for later_minor_version in self.later_minor_versions
+        )
+
+    @property
+    def has_known_statistical_corrections(self) -> bool:
```

Comment:
Add a comment here to describe that it returns true if the version contains a statistical correction – however it may ALSO contain an error?

Reply:
👍